adagrams/game.py

```diff
@@ -1,11 +1,155 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+
+    final_string = []
+    letters_frequency = []
+    
+
+    for letters, frequency in LETTER_POOL.items():
+        for i in range(frequency):
+            letters_frequency.append(letters)
+    
+
+    while len(final_string) < 10:
+        letter_index = random.randint(0, len(letters_frequency) -1)
+
+        if letters_frequency[letter_index] in final_string:
+            continue
+        elif letters_frequency[letter_index] not in final_string:
+            final_string.append(letters_frequency[letter_index])
```

Comment:
Here the letters are only being added once, we would need to change the logic here to where letters could be added multiple times but also respect their frequency limit. Maybe we could remove the letters as the are appended?